modules/ROOT/pages/import-api-specification-exchange.adoc

```diff
@@ -55,8 +55,62 @@ Manage the API specifications linked to Exchange in your project from the API sp
 [NOTE]
 If you want Studio to list your API when searching in Exchange, you must mark your API specification published as an asset as *stable*.
 
+== Implement AsyncAPI Specifications
+
+Studio supports the implementation of AsyncAPI 2.6 specs. When creating an implementation project, Studio imports and scaffolds an AsyncAPI spec hosted on Exchange into an API interface that you can implement.
+
+For supported protocols, see xref:apikit::apikit-asyncapi-module-reference.adoc#supported-message-brokers[Supported Message Brokers] in APIkit for AsyncAPI Module Reference.
+
+=== Access AsyncAPI Implementation Features
+
+. Open Anypoint Code Builder landing page on your Anypoint Platform cloud host (https://anypoint.mulesoft.com/codebuilder/[US] or https://eu1.anypoint.mulesoft.com/codebuilder/[EU]). 
++
+Sign-in is required. The *Terms & Conditions* button is available to the organization administrator only.
+. If the message *AsyncAPI Beta is now available in Anypoint Code Builder and Anypoint Studio* appears on the page, ask your organization administrator to join the Beta program by:
+.. Clicking *Join Beta* from the Anypoint Code Builder landing page.
++
+The *Join Beta* button is available only to your organization administrator.
+.. Reviewing the terms and conditions of the beta program
+.. Clicking *Get Access*
++
+When the AsyncAPI feature is available for your organization, the Anypoint Code Builder landing page provides the following notification:
++
+*AsyncAPI Implementation Beta for Anypoint Code Builder and Studio is enabled for your organization.*
+
+[NOTE]
+You must have access to a supported AsyncAPI specification hosted on Anypoint Exchange. If your AsyncAPI specification is not published to Exchange, see xref:design-center::design-publish.adoc[]. To create an AsyncAPI specification before publishing to Exchange, see xref:design-center::design-async-api.adoc[] in the Design Center documentation.
+
+=== Scaffolding Fundamentals for AsyncAPI Implementations
+
+When scaffolding an AsyncAPI specification into a Mule project (implementation or integration), Studio:
```

Comment:
```suggestion
When scaffolding an AsyncAPI specification into a Mule project, Studio:
```

I don't think Studio docs distinguish these types of projects, so it might be better to just omit the types. Makes it less likely to raise unnecessary questions from customers.

Reply:
OK! Thanks!